Description: Fix build error in src/globalplatformTest.c
 Fix build error in src/globalplatformTest.c due to missing function argument.
Author: Dmitry Eremin-Solenikov <dbaryshkov@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

--- globalplatform-6.0.0.orig/src/globalplatformTest.c
+++ globalplatform-6.0.0/src/globalplatformTest.c
@@ -128,7 +128,7 @@ static OPGP_ERROR_STATUS internal_mutual
 	}
 	status = GP211_mutual_authentication(cardContext, cardInfo, NULL,
 			(PBYTE)OPGP_VISA_DEFAULT_KEY, (PBYTE) OPGP_VISA_DEFAULT_KEY,
-			(PBYTE) OPGP_VISA_DEFAULT_KEY, 0, 0, scp, scpImpl,
+			(PBYTE) OPGP_VISA_DEFAULT_KEY, 0, 0, 0, scp, scpImpl,
 			GP211_SCP01_SECURITY_LEVEL_C_DEC_C_MAC, &securityInfo211);
 	if (OPGP_ERROR_CHECK(status)) {
 		return status;
